tests/ast/data/DataFrame.write.test

```diff
@@ -44,6 +50,12 @@ df.write.save_as_table("saved_table")
 
 df.write.mode("overwrite").save_as_table("saved_table", table_type="temporary")
 
```

Comment:
can we add some repeated calls of `.mode(...)`, `.option(...)`, `.options(...)`? Just to clarify behavior and preserve for the future.